pytest_dbfixtures/factories/postgresql.py

```diff
@@ -176,6 +176,11 @@ def postgresql_proc_fixture(request):
         init_postgresql_directory(
             postgresql_ctl, config.postgresql.user, datadir
         )
+
+
+        with open(os.path.join(datadir, 'pg_hba.conf'), 'a') as f:
```

Comment:
@tomislater, please see https://www.freebsd.org/doc/handbook/jails-ezjail.html

> Inside a jail, access to the loopback address 127.0.0.1 is redirected to the first IP address assigned to the jail.

Thus, on jailed instances pytest-dbfixtures can not connect on the normal loopback. My "fix" will work for all distros, not just FreeBSD (even though it is a freebsd specific fix). 

One can argue about the tradeof regarding security, when the test database is started, everybody will be able to connect.. In a testing environment I dont see this as a problem, but if you think so.. I can take it out of the PR? @fizyk, du you have any feeling about this?